pages/qci_ui_tools.py

```diff
@@ -0,0 +1,44 @@
+from selenium.webdriver.common.by import By
+
+
+class QciUiTools():
```

Comment:
Is there a need to have a separate QciUiTools object? build_qci_spinner_xpath and wait_for_spinner should just be methods in the Page object.  QciUiTools requires a page object at initialization so these methods should just be included in the Page class.